README.md

```diff
@@ -81,69 +81,116 @@ Summary of one-time project setup:
+
+The integration tests provided in this project do not cover all the test cases verified by the unit tests. They verify that the functions work together when invoked correctly. We could classify them as the subset of Integration Tests called *Smoke Tests*, tests that check basic functionality of the application.
+
+For more on different types of software testing, [here is a helpful resource](https://www.atlassian.com/continuous-delivery/software-testing/types-of-software-testing). 
+
+## Code Coverage
+
+It is good practice to check code coverage. Code coverage is a term that is used to describe how much application code is executed when a particular test suite is run. A test suite with a high percentage of coverage is likely to be testing more throughly and have fewer bugs. A code coverage tool can partner with our testing suite to give us a report illustrating the coverage of our tests.
+
+Given that Ada provided all tests cases in this project, we should anticipate high code coverage.
+
+Review the [code coverage exercise](https://github.com/adaGold/code-coverage-exercise) on how to use `pytest-cov` to generate a code coverage report. We will need to change the directory where the application code is located from `student` to `swap_meet`.  
+
+`pytest --cov=swap_meet --cov-report html --cov-report term`
+
+*Note: Code coverage is disabled for integration tests, since unit tests should cover all the code.* [source](https://pypi.org/project/pytest-integration/#:~:text=Overview,integration%20tests%20are%20not%20run)
+
 
 ## Details About How to Run Tests
 
-Run all tests that exist in this project with:
+All the commands described below should be run from the project-root directory `swap_meet`. Note that the project-root directory is the repository `swap-meet`. It is distinct from the directory `swap_meet` that contains the source code.
+
+To run all unskipped tests that exist in this project with:
 
 ```bash
 # Must be in activated virtual environment
 $ pytest
 ```
 
-If you want to run all tests that exist in one file, use:
+To see any `print` statements print to the console, add `-s` to the end of any `pytest` command:
 
 ```bash
 # Must be in activated virtual environment
-$ pytest tests/test_file_name.py
+$ pytest -s
+```
+
+To run all unskipped tests that exist in one file, use:
+
+```bash
+# Must be in activated virtual environment
+$ pytest tests/unit_tests/test_file_name.py
 ```
 
 ... where `test_file_name.py` is replaced with the correct test file name.
 
-If you want to see any `print` statements print to the console, add `-s` to the end of any `pytest` command:
+To run a single test by name:
```

Comment:
I like the gentle reorganization here and adding in running a single test by name.